modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/JavascriptClientCodegen.java

```diff
@@ -1220,4 +1220,12 @@ public void postProcessFile(File file, String fileType) {
             }
         }
     }
+
+    @Override
+    protected String getCollectionFormat(CodegenParameter codegenParameter) {
```

Comment:
@tray2100 Can you please add a 1-liner explaining what this condition does and indicate that `passthrough` is not something officially stated in the openapi spec?

Reply:
Sure that makes sense. I added in the comment. I also rebased and squashed the commits to make things cleaner.